tests/test_app.py

```diff
@@ -1049,6 +1049,167 @@ async def test_bulk_specification_success():
+                    "objects": [
+                        {
+                            "data_type": "gff_metagenome",
+                            "parameters": {
+                                "fasta_file": "fasta_1",
+                                "gff_file": "gff_1",
+                                "genome_name": "mg_1",
+                            },
+                        },
+                        {
+                            "data_type": "gff_metagenome",
+                            "parameters": {
+                                "fasta_file": "fasta_2",
+                                "gff_file": "gff_2",
+                                "genome_name": "mg_2",
+                            },
+                        },
+                    ],
+                },
+            }
+            manifest_2 = "test_manifest_2.json"
+            manifest_2_dict = {
+                "resources": [],
+                "instructions": {
+                    "protocol": "KBase narrative import",
+                    "objects": [
+                        {
+                            "data_type": "gff_genome",
+                            "parameters": {
+                                "fasta_file": "g_fasta_1",
+                                "gff_file": "g_gff_1",
+                                "genome_name": "genome_1",
+                            },
+                        },
+                        {
+                            "data_type": "gff_genome",
+                            "parameters": {
+                                "fasta_file": "g_fasta_2",
+                                "gff_file": "g_gff_2",
+                                "genome_name": "genome_2",
+                            },
+                        },
+                    ],
+                },
+            }
+            with open(base / manifest_1, "w", encoding="utf-8") as f:
+                json.dump(manifest_1_dict, f)
+            with open(base / manifest_2, "w", encoding="utf-8") as f:
+                json.dump(manifest_2_dict, f)
+            resp = await cli.get(f"bulk_specification/?files={manifest_1}  ,   {manifest_2}&dts=1")
+            jsn = await resp.json()
+            # fails for now. will update when schema/parser is properly finished.
+            assert jsn == {
+                "errors": [
+                    {
+                        "type": "cannot_parse_file",
+                        "file": f"testuser/{manifest_1}",
+                        "message": "No import specification data in file",
+                        "tab": None,
+                    },
+                    {
+                        "type": "cannot_parse_file",
+                        "file": f"testuser/{manifest_2}",
+                        "message": "No import specification data in file",
+                        "tab": None,
+                    },
+                ]
+            }
+            # soon will be this:
```

Comment:
This test fails for now, since the parser isn't present yet. When it succeeds, the error check above will be removed and replaced with the real answer.